oio/cli/container/container.py

```diff
@@ -519,7 +519,7 @@ def take_action(self, parsed_args):
         if parsed_args.delete_exceeding_versions:
             system['sys.m2.policy.version.delete_exceeding'] = ''
 
-        if properties:
+        if properties or not system:
```

Comment:
You can have properties and system not set at the same time ?

Reply:
Yes we can. This deletes all the user properties.
```
$ openio container create toto --property test=1 --max-versions 10
+------+---------+
| Name | Created |
+------+---------+
| toto | True    |
+------+---------+
$ openio container show toto
+-----------------+--------------------------------------------------------------------+
| Field           | Value                                                              |
+-----------------+--------------------------------------------------------------------+
| account         | myaccount                                                          |
| base_name       | 8B78B3245B74710F3ACC1BEF4978E621F5E764E01FFB5621D23C4EECA2B7BB3D.1 |
| bytes_usage     | 0B                                                                 |
| container       | toto                                                               |
| ctime           | 1562232773                                                         |
| damaged_objects | 0                                                                  |
| max_versions    | 10                                                                 |
| meta.test       | 1                                                                  |
| missing_chunks  | 0                                                                  |
| objects         | 0                                                                  |
| quota           | Namespace default                                                  |
| status          | Enabled                                                            |
| storage_policy  | Namespace default                                                  |
+-----------------+--------------------------------------------------------------------+
$ openio container unset toto
$ openio container show toto
+-----------------+--------------------------------------------------------------------+
| Field           | Value                                                              |
+-----------------+--------------------------------------------------------------------+
| account         | myaccount                                                          |
| base_name       | 8B78B3245B74710F3ACC1BEF4978E621F5E764E01FFB5621D23C4EECA2B7BB3D.1 |
| bytes_usage     | 0B                                                                 |
| container       | toto                                                               |
| ctime           | 1562232773                                                         |
| damaged_objects | 0                                                                  |
| max_versions    | 10                                                                 |
| missing_chunks  | 0                                                                  |
| objects         | 0                                                                  |
| quota           | Namespace default                                                  |
| status          | Enabled                                                            |
| storage_policy  | Namespace default                                                  |
+-----------------+--------------------------------------------------------------------+
```